electricitymap/contrib/lib/models/events.py

```diff
@@ -371,6 +371,9 @@ def to_dict(self) -> Dict[str, Any]:
             "storage": self.storage.dict(exclude_none=True) if self.storage else {},
             "source": self.source,
             "sourceType": self.sourceType,
+            "correctedModes": list()
+            if self.production is None
+            else list(self.production._corrected_negative_values),
```

Comment:
Any advantage of using list() over []? Just for my own curiosity?

Reply:
Actually this was to create a fully new object, but actually I read online that this is slightly less efficient so I changed it to [] instead, thanks for flagging this.